### DISCUSSION TITLE
Running a script on joining server

### DISCUSSION THREAD

**Discussion by udu3324** [ORelio/Minecraft-Console-Client#1992] [Q&A] [answered]
Is there a possible way to start a script (/script name) automatically on joining a server? I've been looking through the config/readme and can't find anything. My script has "script loop" at the end, but I'm not sure if it will work when rejoining a server.

**Comment by Daenges** [comment, marked as answer]
What you are looking for, is a [task](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-tasks.ini). I also explained them in [my video](https://www.youtube.com/watch?v=JbDpwwETEnU&t=188).
